gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -725,15 +725,19 @@ nnstreamer_filter_find (const char *name)
   if (fw == NULL) {
     /* get sub-plugin priority from ini file and find sub-plugin */
     _str = nnsconf_get_custom_value_string (name, "subplugin_priority");
-    fw = nnstreamer_filter_find_best_fit (_str);
-    g_free (_str);
+    if (_str) {
+      fw = nnstreamer_filter_find_best_fit (_str);
```

Comment:
Q. what was the unnecessary log?

Reply:
nnstreamer_filter_find_best_fit() calls g_strsplit_set(). If str from conf file is null (no group or key in ini file), it will print log about null param.

Reply:
Oh, `g_strsplit_set` prints a log? it's interesting...

Reply:
It's not a special log, g_strsplit_set() internally checks the params using g_return_val_if_fail() :)

I updated condition to check null or empty string in nnstreamer_filter_find_best_fit().